tests/test_my_funcs.py

```diff
@@ -7,13 +8,13 @@ def test_print_hello_world():
 
 
 def test_my_func():
-    mf.my_func(True)
-    mf.my_func(False)
+    my_func(True)
+    my_func(False)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['my_func' may be undefined, or defined from star imports: mypackage.my_funcs](https://app.codacy.com/app/volker.hirthammer/pythonTest/pullRequest?prid=4697838)